google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
@garrettjonesgoogle Rather than doing this, can we handle it the same as IAM: adding the LRO methods directly to the Speech service, and using the reroute feature?